onboard/backend/src/api/mod.rs

```diff
@@ -377,7 +391,7 @@ pub async fn download_game(game_id: String) -> Result<(), Error> {
  * This function will never panic, but contains an `unwrap` call that will never fail. This section
  * is here to make clippy happy.
  */
-pub async fn launch_game(game_id: String) -> Result<JoinHandle<ExitStatus>, Error> {
+pub async fn launch_game(game_id: String) -> Result<(), Error> {
```

Comment:
Is the handle return not important?

Reply:
It's not used anymore, I refactored that stuff all away